src/rsz/test/repair_fanout7_multi.ok

```diff
@@ -11,368 +11,329 @@ worst slack -23.63
+    1350* |       0 |     656 |      360 |      7 |     8 |   +47.7% |   -3.013 |     -305.4 |    150 | load143/D
+    1351* |       0 |     657 |      360 |      7 |     8 |   +47.9% |   -3.013 |     -305.4 |    150 | load143/D
+    1354* |       0 |     659 |      360 |      7 |     8 |   +48.1% |   -3.013 |     -305.4 |    150 | load143/D
+    1356* |       0 |     660 |      360 |      7 |     8 |   +48.2% |   -3.013 |     -305.3 |    150 | load143/D
+    1358* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1359* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1360* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1360* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1361* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1362* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1363* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1364* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1365* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1366* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1367* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1368* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1369* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1370* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1370* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1371* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1372* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1373* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1374* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1375* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    1376* |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
+    final |       0 |     661 |      360 |      7 |     8 |   +48.3% |   -3.013 |     -305.3 |    150 | load143/D
```

Comment:
There is a fair amount of area growth here.  Expected?

Reply:
Not for any concrete reason, no